File: 8KYU/Well of Ideas - Easy Version/solution.cs
using System.Linq;
public class Kata
{
	public static string Well(string[] x)
	{
		var goodCount = x.Where(o => o == "good").Count();
		if(goodCount > 2)
		  return "I smell a series!";
		else if(goodCount > 0 && goodCount <= 2)
		  return "Publish!";
		else
		  return "Fail!";
	}
}

File: 8KYU/Well of Ideas - Easy Version/solution.go
package kata

func Well(x []string) string {
	goodCount := 0

	for i := 0; i < len(x); i++ {
		if x[i] == "good" {
			goodCount = goodCount + 1
		}
	}

	if goodCount > 2 {
		return "I smell a series!"
	} else if goodCount > 0 && goodCount <= 2 {
		return "Publish!"
	} else {
		return "Fail!"
	}
}
